docs/community/dev-card.md

```diff
@@ -0,0 +1,17 @@
+---
+id: dev-card
+title: 'Showcase your open source stats with your dev card'
+sidebar_label: 'Dev Card'
+keywords:
+- dev-card 
+---
+
+The Dev card feature is what we like to call your "open source business card." At the front, you see your username, the number of pull requests you have created, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate.
+[screenshot of front of a dev card](../../static/img/Group%20113.svg)
+
+When you click on the back of your card, you see additional information, including your OpenSauced profile's headline, amount of time you've been using our product, and your [PR velocity rate](../../docs/community/pr-velocity.md).
+[screenshot of back of a dev card](../../static/img/devcard%20vbase%20back.svg).
+
+Whether you are a person who's just beginning their open source journey or a maintainer looking for more contributors and/or sponsors, the Dev Card is a shareable snapshot of your contributions, skills, and engagement in the open source community.
+
+Eager to get started? Check out the [Discovering Open-Source with OpenSauced](https://github.com/open-sauced/intro/blob/main/05-how-to-contribute-to-open-source.md#discovering-open-source-with-opensauced) section in our free beginner course on open source to create your account.
```

Comment:
I think the call to action (CTA) here needs to be, "Eager to get your Dev Card? Create it today and share with your network!"

Reply:
I think, in general, for each of our features that gets documented, the CTA should be for the user to utilize the feature. If we have other materials on how to use it, then linking to that information is good.

Reply:
> I think, in general, for each of our features that gets documented, the CTA should be for the user to utilize the feature. If we have other materials on how to use it, then linking to that information is good.

I like that rule! :) It'll further encourage users to learn how to use the product.